src/test/rustdoc-ui/doc-without-codeblock.rs

```diff
@@ -7,7 +7,7 @@ pub struct Foo;
 /// And then, the princess died.
 //~^ ERROR missing code example in this documentation
 pub mod foo {
-    /// Or maybe not because she saved herself!
+    /// Or maybe not because they saved herself!
```

Comment:
Many examples would make it contextually incorrect e.g. when there is a discussion of a hypothetical entity who IS gendered. It would require a much more careful revision.